database/factories/GenreMovieFactory.php

```diff
@@ -24,8 +24,8 @@ class GenreMovieFactory extends Factory
     public function definition()
     {
         return [
-            'genre_id' => rand(1, 12),
-            'movie_id' => Movie::factory(),
+            'movie_id' => Movie::all()->random()->id,
+            'genre_id' => Genre::all()->random()->id,
```

Comment:
Your suggestion worked very well Vick, there was no need to make a separate seeder and it worked. I did this for all instances where a factory needs to fake a data that is already existing. Let me know what you think Vick XD @vrodriguero